web3/personal/README.md

```diff
@@ -0,0 +1,7 @@
+# web3/personal
```

Comment:
Currently, we have `shhext` package in the root project dir which is also an extension of so-called `web3`. However, I have never seen `web3` package in geth and this is rather exclusively used to web3.js. I would recommend coming up with some different package name.